### ISSUE TITLE
Files that need to be redistributed (Windows)

### ISSUE THREAD

**Issue by JulienMaille** [openvinotoolkit/openvino#1343]
I'm using OpenVino through OpenCV and was wondering what these files are for and if they need to be redistributed on Windows platform:
- cache.json
- pcie-ma248x.elf
- usb-ma2x8x.mvcmd
- usb-ma2450.mvcmd

Thanks a lot

**Comment by jgespino** [comment]
Hi @JulienMaille 

The pcie-ma248x.elf, usb-ma2x8x.mvcmd and usb-ma2450.mvcmd files are the firmware for Myriad devices. You shouldn't need to redistribute these files if you are not using the MYRIAD Plugin. 

Have you taken a look at the [OpenVINO Deployment Manager](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_deployment_manager_tool.html)? It's avaiable for Windows and Linux to help create a deployment package.

Regards,
Jesus

**Comment by JulienMaille** [comment]
@jgespino Thanks for the link. BTW the images showing the interactive mode python script are missing (404)
I have an Intel Compute Stick 2 and can't find this option in the interactive script.
Finally the archive created by that script is a mix of release and debug with both .dll and .lib files, so this is not what I call a redistributable.

**Comment by jgespino** [comment]
Hi @JulienMaille 

You are correct, the Intel Neural Compute Stick 2 (VPU) is not currently available for the Windows version of the Deployment Manager. I will provide your feedback to update the documentation and create a feature request to add the VPU target as an option for Windows as well. 

Regards,
Jesus

**Comment by jgespino** [comment]
Hi @JulienMaille 

Looks like VPU is actually implemented in the Deployment Manager for Windows as well. There is a bug in the configuration file that pointing to the wrong location which hides the VPU target as an option.

As a quick workaround, modify the `C:\Program Files (x86)\IntelSWTools\openvino\deployment_tools\tools\deployment_manager\configs\main.json` file.

Change Line 57, 58, and 59 from:
```
        "deployment_tools/inference_engine/lib/intel64/Release/usb-ma2450.mvcmd",
        "deployment_tools/inference_engine/lib/intel64/Release/usb-ma2x8x.mvcmd",
        "deployment_tools/inference_engine/lib/intel64/Release/pcie-ma248x.elf",
```

To:
```
        "deployment_tools/inference_engine/bin/intel64/Release/usb-ma2450.mvcmd",
        "deployment_tools/inference_engine/bin/intel64/Release/usb-ma2x8x.mvcmd",
        "deployment_tools/inference_engine/bin/intel64/Release/pcie-ma248x.elf",
```

Please give that a try and let me know if this works for you.

Regards,
Jesus

**Comment by JulienMaille** [comment]
Yes it seems to work, thank you!

**Comment by jgespino** [comment]
Great! I will close this issue, feel free to re-open or start a new one if you have additional questions.

**Comment by JulienMaille** [comment]
Fixed in 2021.1
